xls/modules/zstd/BUILD

```diff
@@ -0,0 +1,1068 @@
+        ":dec_mux.v",
+    ],
+    tags = ["manual"],
+)
+
+synthesize_rtl(
+    name = "dec_mux_synth_asap7",
+    standard_cells = "@org_theopenroadproject_asap7sc7p5t_28//:asap7-sc7p5t_rev28_rvt",
+    tags = ["manual"],
+    top_module = "DecoderMux",
+    deps = [
+        ":dec_mux_verilog_lib",
+    ],
+)
+
+benchmark_synth(
+    name = "dec_mux_benchmark_synth",
+    synth_target = ":dec_mux_synth_asap7",
+    tags = ["manual"],
+)
+
+place_and_route(
+    name = "dec_mux_place_and_route",
+    clock_period = "750",
+    core_padding_microns = 2,
+    min_pin_distance = "0.5",
+    placement_density = "0.30",
+    stop_after_step = "global_routing",
+    synthesized_rtl = ":dec_mux_synth_asap7",
+    tags = ["manual"],
+    target_die_utilization_percentage = "10",
+)
+
+xls_dslx_library(
+    name = "dec_demux_dslx",
+    srcs = [
+        "dec_demux.x",
+    ],
+    deps = [
+        ":block_header_dslx",
+        ":common_dslx",
+    ],
+)
+
+xls_dslx_test(
+    name = "dec_demux_dslx_test",
+    library = ":dec_demux_dslx",
+    tags = ["manual"],
+)
+
+xls_dslx_verilog(
+    name = "dec_demux_verilog",
+    codegen_args = {
+        "module_name": "DecoderDemux",
+        "delay_model": "asap7",
+        "pipeline_stages": "2",
+        "reset": "rst",
+        "use_system_verilog": "false",
+    },
+    dslx_top = "DecoderDemux",
+    library = ":dec_demux_dslx",
+    tags = ["manual"],
+    verilog_file = "dec_demux.v",
+)
+
+xls_benchmark_ir(
+    name = "dec_demux_opt_ir_benchmark",
+    src = ":dec_demux_verilog.opt.ir",
+    benchmark_ir_args = {
+        "pipeline_stages": "2",
+        "delay_model": "asap7",
+    },
+    tags = ["manual"],
+)
+
+verilog_library(
+    name = "dec_demux_verilog_lib",
+    srcs = [
+        ":dec_demux.v",
+    ],
+    tags = ["manual"],
+)
+
+synthesize_rtl(
+    name = "dec_demux_synth_asap7",
+    standard_cells = "@org_theopenroadproject_asap7sc7p5t_28//:asap7-sc7p5t_rev28_rvt",
+    tags = ["manual"],
+    top_module = "DecoderDemux",
+    deps = [
+        ":dec_demux_verilog_lib",
+    ],
+)
+
+benchmark_synth(
+    name = "dec_demux_benchmark_synth",
+    synth_target = ":dec_demux_synth_asap7",
+    tags = ["manual"],
+)
+
+place_and_route(
+    name = "dec_demux_place_and_route",
+    clock_period = "750",
+    core_padding_microns = 2,
+    min_pin_distance = "0.5",
+    placement_density = "0.30",
+    stop_after_step = "global_routing",
+    synthesized_rtl = ":dec_demux_synth_asap7",
+    tags = ["manual"],
+    target_die_utilization_percentage = "5",
+)
+
+xls_dslx_library(
+    name = "block_dec_dslx",
+    srcs = [
+        "block_dec.x",
+    ],
+    deps = [
+        ":common_dslx",
+        ":dec_demux_dslx",
+        ":dec_mux_dslx",
+        ":raw_block_dec_dslx",
+        ":rle_block_dec_dslx",
+    ],
+)
+
+xls_dslx_test(
+    name = "block_dec_dslx_test",
+    library = ":block_dec_dslx",
+    tags = ["manual"],
+)
+
+xls_dslx_verilog(
+    name = "block_dec_verilog",
```

Comment:
curious if those are still needed (now that you have a verilog target for the top-level)